wpilibcIntegrationTests/src/CircularBufferTest.cpp

```diff
@@ -5,12 +5,11 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "CircularBuffer.h"
+#include <array>
 
+#include "CircularBuffer.h"
```

Comment:
What should I do about FilterNoiseTest.cpp and FilterOutputTest.cpp with LinearDigitalFilter.cpp?